tweetapi/serializer.py

```diff
@@ -1,27 +1,28 @@
+from asyncore import write
 from rest_framework import serializers
 from .models import ProfileModel, TweetModel,LikeModel,RetweetModel,CommentModel
 from django.contrib.auth.models import User
 
 
+class ProfileSerilizer(serializers.ModelSerializer):
+    first_name = serializers.PrimaryKeyRelatedField(read_only=True)
+    last_name = serializers.PrimaryKeyRelatedField(read_only=True)
```

Comment:
This is confusing. you already have it in fields

see https://www.django-rest-framework.org/api-guide/relations/#primarykeyrelatedfield